### PR TITLE
[WIP] Adds Publishing guidance to manual

### PR DESCRIPTION
**Pull request by ljhwang**
Initial pull request to add Publishing guidance to the ASPECT manual.

This is labeled WIP as it is dependent on adding specific guidance to the ASPECT webpage. @gassmoeller and I discussed that specific guidance should be added similarly to How to Cite? is implemented.  Hence @tjhei (?), additional work needs to be done to add a webpage with the specific guidance specified here for ASPECT: https://docs.google.com/document/d/1MOr5Zh1uZYxQs_B3YHMTHQBvc44Vrv3_gpVftTGc6rw/edit

When the new webpage is implemented, the manual should be updated with the correct link along with any other edits needed.

Thanks.

### CODE REVIEW

File: doc/sphinx/user/intro.md [hunk 1]
```diff
@@ -56,6 +56,19 @@ For what exactly to cite and suggestions for acknowledgments, please see **<http
 
 Also see {cite:t}`aspectmanual,aspect-doi-v1.5.0,aspect-doi-v2.0.0,aspect-doi-v2.0.1,kronbichler:etal:2012,heister:etal:2017`.
 
+## Publishing with ASPECT
+Open research statements are now a common requirement when publishing research. These support reuse, validation, and citation and often take the form of *Data availability, Data access, Code availability, Open Research*, and *Software availability* statements.
+ We recommend depositing input files that allow your published research to be reproduced and output model data in support of your research outcomes and figures. In addition, consider depositing model files that may be reused by others.
```
Comment (bangerth) [inline doc/sphinx/user/intro.md:61]: Depositing where?

File: doc/sphinx/user/intro.md [hunk 2]
```diff
@@ -56,6 +56,19 @@ For what exactly to cite and suggestions for acknowledgments, please see **<http
...
+
+
+Remember to cite software and data in your text as well as in your Data Availability or similar statement.
+
+File should be deposited in an approved repository.
```
Comment (bangerth) [inline doc/sphinx/user/intro.md:66]: This sentence belongs to the end of the first paragraph you add. But it would still be useful to explain what "approved repository" is supposed to be.

File: doc/sphinx/user/intro.md [hunk 3]
```diff
@@ -56,6 +56,19 @@ For what exactly to cite and suggestions for acknowledgments, please see **<http
...
+
+Specifc guidance in meeting publisher’s data availability requirements for ASPECT is available here: **insert aspect website here**.
```
Comment (bangerth) [inline doc/sphinx/user/intro.md:68]: Might want to fill this in :-)

**Review by bangerth — CHANGES_REQUESTED** [review]

File: doc/sphinx/user/intro.md (hunk 2 above)
Comment (ljhwang) [reply doc/sphinx/user/intro.md:66]: This statement is a follow on to you comment on line 61 .  Line 70 is the url to the CIG website which has more explicit wording.  I will modify to make the links between the statements stronger.

**Comment by ljhwang** [comment]
I have modified the text to address previous comments and to align with approach as discussed in the User meeting.  

Text now refers to the CIG website for guidance on approved repositories and the ASPECT website 'How to Cite' for suggested data availability statement.

@tjhei 
For[ aspect.geodynamics.org ](https://aspect.geodynamics.org/citing.html?src=www&ver=release) Step 5, you can find the text to be added here: https://docs.google.com/document/d/1MOr5Zh1uZYxQs_B3YHMTHQBvc44Vrv3_gpVftTGc6rw/edit#bookmark=id.v2sajf86x9nf  and below:

>>>
Step 5: Add the following text to your data availability statement
Deposit your data in an approved repository which will assign a PID e.g. DOI and enable citation to your data set - Authors, ZZZZ. See https://geodynamics.org/software/software-bp/software-publishing for guidance on approved repositories.

The input parameter, data and log files used for the models in the study are available at PID (Authors, ZZZZ) under [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/).

ASPECT 2.5.0 ( \cite{heister:etal:2017,kronbichler:etal:2012,aspect-doi-v2.5.0,aspectmanual}) used in these models is freely available under the GPL v2.0 or later license for download through its software landing page https://geodynamics.org/resources/aspect or 10.5281/ZENODO.8200213 . The project is being actively developed on GitHub and can be accessed via https://github.com/geodynamics/aspect.

**Comment by tjhei** [comment]
> For[ aspect.geodynamics.org ](https://aspect.geodynamics.org/citing.html?src=www&ver=release) Step 5

Should this maybe replace step 2 then? It seems silly to have both.

**Comment by ljhwang** [comment]
Since the Data Availability is appended at the end of an article, perhaps this should be Step 4 and adding the references moved to step 5

**Comment by tjhei** [comment]
Do you agree that step 2 is redundant? I will try out what that looks like and show you.

**Comment by tjhei** [comment]
how about this? 
[aspect-cite-preview.pdf](https://github.com/geodynamics/aspect/files/13058276/aspect-cite-preview.pdf)

**Comment by ljhwang** [comment]
I am wondering if the preamble is too wordy. Maybe get rid of all of the text between the ()'s and also link to the manual???

Otherwise it looks great.  

Thanks.

**Comment by bangerth** [comment]
@ljhwang Where does this currently stand? Do you want to make more changes?

**Comment by ljhwang** [comment]
What is merged looks good to me. I have gone and ahead and closed this.
